python/paddle/fluid/layers/nn.py

```diff
@@ -4844,7 +4844,7 @@ def autoincreased_step_counter(counter_name=None, begin=1, step=1):
     return counter
 
 
-def reshape(x, shape, actual_shape=None, act=None, inplace=True, name=None):
+def reshape(x, shape, actual_shape=None, act=None, inplace=False, name=None):
```

Comment:
Another question. should `inplace` be decided by memory optimizer? So, it's not decided by the API

Reply:
Currently, the memory optimizer can't reuse the input variable directly. We will refine it later.